src/layers/selayers.jl

```diff
@@ -0,0 +1,46 @@
+"""
+    squeeze_excite(inplanes; reduction = 16, rd_divisor = 8,
+                   activation = relu, gate_activation = sigmoid, norm_layer = identity,
+                   rd_planes = _round_channels(inplanes ÷ reduction, rd_divisor, 0.0))
+
+Creates a squeeze-and-excitation layer used in MobileNets and SE-Nets.
+
+# Arguments
+
+  - `inplanes`: The number of input feature maps
+  - `reduction`: The reduction factor for the number of hidden feature maps
+  - `rd_divisor`: The divisor for the number of hidden feature maps.
+  - `activation`: The activation function for the first convolution layer
+  - `gate_activation`: The activation function for the gate layer
+  - `norm_layer`: The normalization layer to be used after the convolution layers
+  - `rd_planes`: The number of hidden feature maps in a squeeze and excite layer
+"""
+function squeeze_excite(inplanes; reduction = 16, rd_divisor = 8,
```

Comment:
Does this not break MobileNet? The interface has changed.

Reply:
Nope. `MobileNet` uses `invertedresidual`, not `squeeze_excite` directly. I made sure to change `invertedresidual` so that there would be no breaks